### PR TITLE
Refactor NodeJs based KeyCloak management scripts

### PR DESCRIPTION
**Pull request by ShellyXueHan**
This PR merges the script that `import LDAP users to keycloak` to the nodejs based management folder. Also update the original scripts to handle edge cases when importing users.

Note: this refactoring also include updates from PR https://github.com/BCDevOps/keycloak-admin/pull/3 LDAP access is not tested yet as we lack proper credential at the moment.

### CODE REVIEW

File: realm-management/libs/import-users.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+/**
+ * Import LDAP users to file with GUID attributes
+ * TODO: turn ldap requests into async and await functions
+ */
+const importLdapUsers = async () => {
+  try {
+    /* Read users.json file */
+    const users = JSON.parse(
+      fs.readFileSync(LDAP_USERS_FILE_PATH, { encoding: 'utf-8' }, (err) => {
```
Comment (patricksimonian) [inline realm-management/libs/import-users.js:15]: doesn't readFileSync return a `Buffer` and not a `String`?

File: realm-management/libs/import-users.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        return new ldap.EqualityFilter({
+          attribute: 'samAccountName',
+          value: user.username,
+        });
+      }),
+    });
+
+    // create ldap client:
+    console.log('create ldap client');
+    const client = ldap.createClient({
```
Comment (patricksimonian) [inline realm-management/libs/import-users.js:39]: is this a sync or async function?

File: realm-management/libs/import-users.js (hunk 1 above)
Comment (ShellyXueHan) [reply realm-management/libs/import-users.js:15]: a buffer, so need to json parse it

File: realm-management/libs/import-users.js (hunk 2 above)
Comment (ShellyXueHan) [reply realm-management/libs/import-users.js:39]: it's sync originally, plan to turn it to async. but I'm waiting for some ldap credential to test it out first (see TODO comments). btw this is based on https://github.com/BCDevOps/keycloak-admin/pull/3

**Review by patricksimonian — APPROVED** [review]

**Pull request merged**